React/my-app/src/screens/profile/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+import Profile from "./layout";
+import { connect } from "react-redux";
+import { getUserInfo, resetProfileView } from "../../redux/accounts/actions";
+
+class ProfileContainer extends Component {
+  state = {
+    emailError: null,
+    passwordError: null,
+    redirect: false
+  };
+
+  componentWillMount() {
```

Comment:
Use `componentDidMount` to do async operations in all components.
https://reactjs.org/docs/react-component.html#componentwillmount
https://reactjs.org/docs/react-component.html#componentdidmount